### ISSUE TITLE
Validation-rectified datasets require rectified file-naming for postprocessing

### ISSUE THREAD

**Issue by TonyB9000** [E3SM-Project/esgfpub#67]
Where irregular restarts or partial re-runs demand validation rectification of overlapping datasets, the current methods involve the generation of dataset files with filenames <name-date>(n).nc, or <name-date>.trunc.nc, whereas ncclimo and related postprocess tools demand the uniform <name-date>.nc

**Comment by TonyB9000** [comment]
Originally, it was "fixed" by running a "unify-naming" script that forced all of the data files to have the structure of the very first file.  That was Sterling's methodology (but I created the script to automate).  At that time, some process would balk if the naming was variant (maybe esgpublish, but this changed with the subsequent versions).  It might have been Sterling's own "timeseries" cwltool workflow.   I think the term "fixed" might be premature.  Recently, for use as input to postprocessing, I used a similar technique - but instead of renaming the actual files, I created a (superior-version) directory of symlinks with uniform names, (in the pub_dir), and once postprocess/e2c generation was completed, that (superior-version) directory was manually deleted.  Not exactly a "fix"...
